dss/util/__init__.py

```diff
@@ -438,8 +438,8 @@ def _get_jwt_from_service_account_credentials(self):
                    'exp': exp,
                    'email': service_credentials["client_email"],
                    'scope': ['email', 'openid', 'offline_access'],
-                   'https://auth.data.humancellatlas.org/group': 'hca',
-                   'https://auth.data.humancellatlas.org/email': service_credentials["client_email"]
+                   'group': 'dss',
```

Comment:
this will cause breaking changes in the DSS, we should change the security group on /put-file and /put subscriptions. and/or make it configurable.